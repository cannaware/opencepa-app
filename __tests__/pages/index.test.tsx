import renderer from 'react-test-renderer';
import IndexPage from '../../pages/index';

describe('Index page', () => {
  it('should match the snapshot', () => {
    const launch = {
      flight_number: 110,
      details:
        "SpaceX's 21st ISS resupply mission on behalf of NASA and the first under the CRS-2 contract, this mission brings essential supplies to the International Space Station using the cargo variant of SpaceX's Dragon 2 spacecraft. The external payload for this mission is the Nanoracks Bishop Airlock. Falcon 9 and Dragon launch from LC-39A, Kennedy Space Center and the booster is expected to land on an ASDS. The mission will be complete with return and recovery of the Dragon capsule and down cargo.",
      launch_year: '2020',
    };
    const tree = renderer.create(<IndexPage launch={launch} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
