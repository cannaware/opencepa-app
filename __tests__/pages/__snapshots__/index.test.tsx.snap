// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<main>
  <h1>
    Next SpaceX Launch: 
    110
  </h1>
  <p>
    SpaceX's 21st ISS resupply mission on behalf of NASA and the first under the CRS-2 contract, this mission brings essential supplies to the International Space Station using the cargo variant of SpaceX's Dragon 2 spacecraft. The external payload for this mission is the Nanoracks Bishop Airlock. Falcon 9 and Dragon launch from LC-39A, Kennedy Space Center and the booster is expected to land on an ASDS. The mission will be complete with return and recovery of the Dragon capsule and down cargo.
  </p>
  <p>
    2020
  </p>
  <p>
    <a
      href="/info"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Ir a Info.
    </a>
  </p>
</main>
`;
